src/index.scss

```diff
@@ -61,3 +61,7 @@ body {
     background-color: $light-100;
   }
 }
+
+mark {
+  padding: 0;
+}
```

Comment:
Interesting.. are we using paragon's `DocSearch-Container` ? Because it already has `padding: 0`, cf 
https://github.com/openedx/paragon/blob/4b63a24101e53d156a66d5ad1c28190eb1767d49/www/src/components/Search/Search.scss#L149C7-L153

(But no worries on this change; refactoring that isn't in scope here.)

Reply:
> Interesting.. are we using paragon's DocSearch-Container ?

No. This seems to be used in the Paragon site, but not sure if we have a paragon component for it. We could take a look at this later.

I forgot to point out why I made this. This is not related to the main change in the PR. Let me know if you think this belongs to another PR.
Before:
![Screen Shot 2024-09-06 at 13 15 14](https://github.com/user-attachments/assets/bc45bd69-b5a4-403a-a507-b6895bf59210)
After:
![Screen Shot 2024-09-06 at 13 16 53](https://github.com/user-attachments/assets/5d715739-660d-48d5-a8e2-44b479d92b9f)

Reply:
I think it's ok to include this change here, it's rly minor.